packages/operator/pkg/apiserver/routes/v1/collector.go

```diff
@@ -0,0 +1,119 @@
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+//
+
+package v1
+
+import (
+	"database/sql"
+	"errors"
+	"github.com/gin-gonic/gin"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/apiserver/routes"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/apiserver/routes/v1/configuration"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/apiserver/routes/v1/connection"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/apiserver/routes/v1/deployment"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/apiserver/routes/v1/packaging"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/apiserver/routes/v1/training"
+	userinfo "github.com/odahu/odahu-flow/packages/operator/pkg/apiserver/routes/v1/user"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/config"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/repository/connection/kubernetes"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/repository/connection/vault"
+	conn_service "github.com/odahu/odahu-flow/packages/operator/pkg/service/connection"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/utils"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	deploy_kube_client "github.com/odahu/odahu-flow/packages/operator/pkg/kubeclient/deploymentclient"
+	pack_kube_client "github.com/odahu/odahu-flow/packages/operator/pkg/kubeclient/packagingclient"
+	train_kube_client "github.com/odahu/odahu-flow/packages/operator/pkg/kubeclient/trainingclient"
+
+	conn_repo_type "github.com/odahu/odahu-flow/packages/operator/pkg/repository/connection"
+	deploy_repo "github.com/odahu/odahu-flow/packages/operator/pkg/repository/deployment/postgres"
+	pack_repo "github.com/odahu/odahu-flow/packages/operator/pkg/repository/packaging/postgres"
+	train_repo "github.com/odahu/odahu-flow/packages/operator/pkg/repository/training/postgres"
+)
+
+func SetupV1Routes(routeGroup *gin.RouterGroup, kubeMgr manager.Manager, db *sql.DB, cfg config.Config) (err error) {
+
+	k8sClient := kubeMgr.GetClient()
+	k8sConfig := kubeMgr.GetConfig()
+
+	var connRepository conn_repo_type.Repository
+	switch cfg.Connection.RepositoryType {
+	case config.RepositoryKubernetesType:
+		connRepository = kubernetes.NewRepository(
+			cfg.Connection.Namespace,
+			k8sClient,
+		)
+	case config.RepositoryVaultType:
+		var err error
+		connRepository, err = vault.NewRepositoryFromConfig(cfg.Connection.Vault)
+		if err != nil {
+			return err
+		}
+	default:
+		return errors.New("unexpect connection repository type")
+	}
+
+
+	toolchainRepo := train_repo.ToolchainRepository{DB: db}
+	trainRepo := train_repo.TrainingPostgresRepo{DB: db}
+	piRepo := pack_repo.PackagingIntegrationRepository{DB: db}
+	packRepo := pack_repo.PackagingPostgresRepo{DB: db}
+	deployRepo := deploy_repo.DeploymentPostgresRepo{DB: db}
```

Comment:
_Minor by still..._
I'd suggest to unify the naming here: either all of these repos has "Postgres" in a name or all of them don't. Or maybe we could replace "Postgres" by "DB" which is shorter and saves the sense.